test/Model.spec.js

```diff
@@ -496,4 +496,30 @@ describe("Model", () => {
             });
         });
     });
+
+    describe("set and get", () => {
+        it("should throw an error on invalid input when calling Model:get", (done) => {
+            const model = new Model();
+            model.get("{\"foobar\":[]}").subscribe(() => {
+                done(new Error("Did not receive an error when one was expected"));
+            }, (e) => {
+                console.log(e);
+                expect(e instanceof Error).toBe(true);
+                expect(e.message.indexOf("Path syntax validation error")).toBeGreaterThan(-1);
```

Comment:
You could use an inline snapshot here, it would provide better feedback in case of failure:
https://jestjs.io/docs/expect#tomatchinlinesnapshotpropertymatchers-inlinesnapshot
https://jestjs.io/docs/snapshot-testing#inline-snapshots